Frends.OAuth.CreateJWTToken/Frends.OAuth.CreateJWTToken/CreateJWTToken.cs

```diff
@@ -42,6 +43,11 @@ public static TokenResult CreateJWTToken(Input input)
                 CryptoProviderFactory = new CryptoProviderFactory { CacheSignatureProviders = false }
             };
         }
+        else
+        {
+            ecdsa.ImportFromPem(input.PrivateKey);
+            signingCredentials = new SigningCredentials(new ECDsaSecurityKey(ecdsa), input.SigningAlgorithm.ToString());
+        }
         return new TokenResult(CreateToken(signingCredentials, input, isSymmetric));
     }
 
```

Comment:
https://github.com/FrendsPlatform/Frends.OAuth/blob/05a1fbcdc60258adde54047bdb0e99f2409894d8/Frends.OAuth.CreateJWTToken/Frends.OAuth.CreateJWTToken/CreateJWTToken.cs#L109-L115
Are these necessary? I think the method never goes to these because the isSymmetric is check first.